lib/components/WidthProvider.jsx

```diff
@@ -77,7 +83,7 @@ export default function WidthProvider<Config>(
     onWindowResize = () => {
       if (!this.mounted) return;
       // eslint-disable-next-line react/no-find-dom-node
-      const node = ReactDOM.findDOMNode(this); // Flow casts this to Text | Element
+      const node = this.currentNode.current; // Flow casts this to Text | Element
```

Comment:
I don't think this will work unless we forward the underlying actual DOM ref from the underlying Grid Layout.  This is going to be a `React.Ref<typeof ReactGridLayout>`, not `React.Ref<HTMLElement>`.